docs/community/usingtokens.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Using your UMA tokens
+sidebar_label: How to use your UMA tokens?
+---
+
+## Voting
+
+The UMA token is primarily a governance token, which allows token holders to vote on UMA Improvement Proposals (UMIPs) and on price requests made to the DVM through our [Voter Dapp](https://vote.umaproject.org/).   
+
+Those who vote with the majority consensus are granted 0.05% inflationary rewards for participating in governance and for helping to keep our oracle secure.  Upcoming UMIPs can be tracked through our Discourse, while DVM votes are discussed on the #voting channel in the UMA Discord.  If you are new to voting on UMA take a look at our [video walkthrough](https://www.loom.com/share/ed44dbea28af4124b40af17d4c69e31c) of our voter Dapp.
+
+## Yield Dollar Minting
+
+UMA tokens are accepted as collateral for minting Yield Dollars on the [UMAnji platform](https://umanji.xyz/) from Machina Labs.  UMA tokens are deposited as collateral to mint the yUMA21 token, which expires to exactly $1 on 31st December 2021. 
+
+Liquidity mining rewards are available for those who mint the token and use it to provide liquidity in the [yUMA21/USDC](https://info.uniswap.org/token/0x3108c33b6fb38efedaefd8b5f7ca01d5f5c7372d) liquidity pool on Uniswap.
+
+## Call Options Minting
+
+A call option gives the buyer the right - but not the obligation - to buy an asset at a specified price (strike) at a specific time period (expiry).  Call Options for UMA can be minted on the [MySynths platform](http://mysynths.finance). 
+
+Call Options may be an unfamiliar product to many DeFi users, so we would encourage people to read the articles explaining call options and how they have been implemented on UMA.  For a deeper dive, there is also an internal learning session on our YouTube which goes into more depth. 
```

Comment:
insert links to Kevin's medium articles as well as the internal learning session on youtube.